### PR TITLE
Add support for ashley 1.0.0-beta.3 + release 5.22.0

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

We want to deploy [ashley 1.0.0-beta.3](https://github.com/openfun/ashley/releases/tag/v1.0.0-beta.3) with arnold.
Since it introduced new settings and a search feature based on elasticsearch, some adjustments needs to be done in `ashley` arnold application.

Also, Ashley needs to be deployed with elasticsearch 5.x
A bug in the `elasticsearch` application configuration prevents from deploying it in its version 5.x.

## Proposal

- [x] Add ansible variables to configure `elasticsearch` for ashley
- [x] Add cronjob in `ashley` to update the search index
- [x] Fix elasticsearch configuration (unable to deploy a 5.x elasticsearch with arnold)
- [x] Prepare arnold release 5.22.0

### CODE REVIEW

File: apps/ashley/vars/all/main.yml [hunk 1]
```diff
@@ -37,6 +37,12 @@ ashley_django_settings_module: "settings"
 ashley_django_configuration: "Development"
 ashley_secret_name: "ashley-{{ ashley_vault_checksum | default('undefined_ashley_vault_checksum') }}"
 
+# -- haystack / elasticsearch settings
+ashley_elasticsearch_host: "elasticsearch"
+ashley_elasticsearch_port: "9200"
+ashley_elasticsearch_index: "ashley"
+ashley_cron_reindex_schedule: "17 4,12,20 * * *"
```
Comment (jmaupetit) [inline apps/ashley/vars/all/main.yml]: Can you explain or add a comment to justify this frequency?

**Review by lunika — APPROVED** [review]

**Review by jmaupetit — APPROVED** [review]

File: apps/ashley/vars/all/main.yml (hunk 1 above)
Comment (madmatah) [reply apps/ashley/vars/all/main.yml]: Oh, I had our use case in mind when I was writing this.
Maybe I should set `0 * * * *` as a default value to launch it every hour + a comment. It's easier to understand.
WDYT @jmaupetit  ?

Comment (jmaupetit) [reply apps/ashley/vars/all/main.yml]: Yes !

**Pull request merged**